combat-trainer.lic

```diff
@@ -1912,6 +1931,14 @@ class GameState
     @aim_queue.shift
   end
 
+ def next_combo_action
```

Comment:
Do you mean both the aim_queue.shift and the Analyze_combo_array? def next_combo_action would be a new function that I included to follow convention, I can just remove it from the commit.